src/components/clusterWizard/ClusterDetails.tsx

```diff
@@ -274,6 +249,18 @@ const ClusterDetailsForm: React.FC<ClusterDetailsFormProps> = (props) => {
                       isRequired
                     />
                   )}
+                  {isSingleNodeOpenshiftEnabled && (
```

Comment:
Let's merge this after release and rebase on top of #460

Reply:
resolved